src/Mono.Android/Android.App/Activity.cs

```diff
@@ -12,6 +12,20 @@ partial class Activity {
 			return this.FindViewById (id)!.JavaCast<T> ();
 		}
 
+#if NET7_0_OR_GREATER || (NET6_0_OR_GREATER && ANDROID_33) || ANDROID_34
+		// See: https://cs.android.com/android/platform/superproject/+/master:frameworks/base/core/java/android/app/Activity.java;l=3430
+		public T RequireViewById<T> (int id)
+			where T : Android.Views.View
+		{
+			var view = FindViewById<T> (id);
+			if (view == null)
+			{
+				throw new Java.Lang.IllegalArgumentException ($"ID 0x{id:X} does not reference a View of type '{typeof (T)}' inside this Activity");
```

Comment:
As this exception type *doesn't* have a way to mention the parameter name, should this message instead mention it?

```csharp
"Could not find a View for parameter `id` with value 0x{id:X}!"
```

Reply:
I used the exact same text on:

https://cs.android.com/android/platform/superproject/+/master:frameworks/base/core/java/android/app/Activity.java;l=3430

But added the hex value & type.

Maybe I just change `ID` to:

```suggestion
				throw new Java.Lang.IllegalArgumentException ($"Parameter 'id' of value 0x{id:X} does not reference a View of type '{typeof (T)}' inside this Activity");
```